files/build_templates/sonic_debian_extension.j2

```diff
@@ -297,6 +298,14 @@ fi
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-device-data_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+{% if sonic_asic_platform == "vs" %}
+for PLATFORM in `ls ./device/virtual`; do
+    if [ -f ./device/virtual/$PLATFORM/asic.conf ]; then
+        echo "NUM_ASIC=$NUM_ASIC" | sudo tee $FILESYSTEM_ROOT_USR_SHARE_SONIC_DEVICE/$PLATFORM/asic.conf
```

Comment:
Why sudo?
The whole repo is owned by current user. But this file is owned by root. 
Is this really needed?